python-sdk/examples/opencv/opencv.py

```diff
@@ -86,13 +90,13 @@ def main():
 
 
 def clean():
-    client = Client()
+    client = Client.from_config()
     client.pps.delete_pipeline(pipeline=pps.Pipeline(name="montage"))
     client.pps.delete_pipeline(pipeline=pps.Pipeline(name="edges"))
 
     client.pfs.delete_repo(repo=pfs.Repo.from_uri("images"), force=True)
 
 
 if __name__ == "__main__":
-    # clean()
+    clean()
```

Comment:
If we are going to incorporate this clean function, we should probably name the resources something more unique? I'm envisioning us accidentally deleting someones "images" repo. maybe prepend `opencv-` to the resource names?

Reply:
unfortunately edges.py assumes that the input is `/pfs/images` so unless we change that too then the example won't work